### DISCUSSION TITLE
pv.Spline command  in Pyansys using spaceclaim mode

### DISCUSSION THREAD

**Discussion by santoshdoddipatla** [ansys/pyansys-geometry#984] [Q&A]
first i created list of coordinate points of airfoil then create a plane
i use below code to create a airfoil using pyvista spline command

airfoil_points = np.array(upper + lower[::-1])

spline=Sketch(plane)
(pv.Spline(airfoil_points)
)
spline.plot()

code is executing but plot is not visible, why?.....

![Screenshot 2024-02-11 180602](https://github.com/ansys/pyansys-geometry/assets/143094710/f252df03-7b7b-42d2-9166-ff00baa4af86)

i want to extrude  that spline is it possible?

**Comment by RobPasMue** [comment]
Hi @santoshdoddipatla - support for Splines is not implemented in PyAnsys Geometry **yet**. There is an issue #46 for tracking this implementation. It does not only depend on being able to sketch it using the ``Sketch`` API (which relies on PyVista) but it also implies being able to extrude the given spline which is currently not possible from the Pythonic API. We will work on this since it is in our todo list but I am sorry for any inconveniences. In the meantime, you could approximate your profile using straight lines or approximated curves. We are doing this for example in the ``ansys.geometry.core.sketch.gears`` module.

**Comment by RobPasMue** [comment]
Hey @santoshdoddipatla - I know it is not the Spline implementation what I am going to show but... if you are interested in building an airfoil, have a look at this example!

https://geometry.docs.pyansys.com/version/dev/examples/04_applied/01_naca_airfoils.html
